packages/cli-e2e/setup.ts

```diff
@@ -22,6 +23,8 @@ async function clearChromeBrowsingData(browser: Browser) {
 
 export default async function () {
   mkdirSync(SCREENSHOTS_PATH, {recursive: true});
+  process.env.GITHUB_ACTION =
+    process.env.GITHUB_ACTION || randomBytes(16).toString('hex');
```

Comment:
@louis-bompart I think we will end up with issues if we don't convert to alphanumerical characters only.

Reply:
True, good catch!